src/components/blog/BlogOverview.jsx

```diff
@@ -1,5 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Container, Row } from "react-bootstrap";
+import SingleBlogCard from "./SingleBlogCard";
+import "./style.overview.scss";
+import BlogHeader from "./BlogHeader";
 
 export default function BlogOverview() {
-  return <div></div>;
+  const [allBlogs, setAllBlogs] = useState([]);
+  const [blogsImages, setBlogsImages] = useState([]);
+
+  // To all of our data from wordpress
+
+  const getAllBlogsAndImages = async () => {
+    const blogs = await fetch("http://eecommerce.local/wp-json/wp/v2/health");
+    const blogsMedia = await fetch(
+      "http://eecommerce.local/wp-json/wp/v2/media"
```

Comment:
is this url a local link? please make it a real link to the wp api

Reply:
We are using WordPress locally  because we didn't host our WordPress website ,should we host the website before pull requesting again ?

Reply:
yes, please use wordpress to create a free site and use the api that way